logging/logging.go

```diff
@@ -687,6 +687,15 @@ type HTTPRequest struct {
 	// validated with the origin server before being served from cache. This
 	// field is only meaningful if CacheHit is true.
 	CacheValidatedWithOriginServer bool
+
+	// The number of HTTP response bytes inserted into cache. Set only when a cache fill was attempted.
+	CacheFillBytes int64
```

Comment:
Where do these fields get set? They are read in `fromHTTPREquest` below, but they don't seem to be written.

Reply:
Those fields are public to clients of library. They are not supposed to be set here.